bakerydemo/base/fixtures/bakerydemo.json

```diff
@@ -1444,6 +1365,13 @@
       "workflow": 1
     }
   },
+  {
+    "model": "wagtailcore.workflowcontenttype",
```

Comment:
From what I can tell this is a new model from a minor wagtail release that just hasn't been included until now